libs/blocks/quiz/utils.js

```diff
@@ -378,7 +414,7 @@ export const getAnalyticsDataForBtn = (selectedQuestion, selectedCards) => {
   }
 };
 
-const getAnalyticsDataForLocalStorage = (answers) => {
+export const getAnalyticsDataForLocalStorage = (answers) => {
```

Comment:
I see a bunch of `export` declarations, but no files where these are imported. Should they actually be exported in the first place?

Reply:
Export statements are for the upcoming PR for the UT for this file. No effect for this file tbh.